documentation/asciidoc/topics/proc_create_cluster_auth.adoc

```diff
@@ -16,7 +15,8 @@ link:#ref_auth_secret-auth[Authentication Secrets].
 $ {oc_apply_cr} connect_secret.yaml
 ----
 +
-. Modify your custom resource definition to include the authentication secret and then apply the changes.
+. Specify the authentication secret with the `spec.security.endpointSecretName`
+CR object and then apply the changes.
```

Comment:
Same as above. There is nothing like `spec.security.endpointSecretName` CR object